### DISCUSSION TITLE
Erasure Encoding

### DISCUSSION THREAD

**Discussion by codeninja75** [minio/minio#14192] [Q&A] [answered]
Any thoughts on adding in Erasure coding support for min.io?  We've got some other storage solutions that offer erasure encoding and it seems to offer much lower overhead and thus more storage utilization.  https://en.wikipedia.org/wiki/Erasure_code

**Comment by harshavardhana** [comment, marked as answer]
> Any thoughts on adding in Erasure coding support for min.io? We've got some other storage solutions that offer erasure encoding and it seems to offer much lower overhead and thus more storage utilization. https://en.wikipedia.org/wiki/Erasure_code

Erasure coding has been supported since Day 1 https://docs.min.io/docs/distributed-minio-quickstart-guide.html

**Reply by codeninja75** [reply]
Thank you!  I clearly overlooked that in the docs.
